how-to/how-to-use-kata-containers-with-acrn.md

```diff
@@ -0,0 +1,126 @@
+
+# Kata Containers with ACRN
+
+- [Introduction](#introduction)
+- [Pre-requisites](#pre-requisites)
+- [Install and Configure Docker](#install-and-configure-docker)
+- [Configure Kata Containers with ACRN](#configure-kata-containers-with-acrn)
+- [ACRN Reference](#acrn-reference)
+
+## Introduction
+
+ACRN is a flexible, lightweight reference hypervisor, built with real-time and safety-criticality in mind and optimized to streamline embedded development through an open-source platform.
+
+There’re two key components of ACRN: the hypervisor itself and the ACRN device model. The __ACRN Hypervisor__ is a Type 1 reference hypervisor stack, running directly on bare-metal, and is suitable for a variety of IoT and embedded device solutions.  ACRN hypervisor addresses the gap that currently exists between datacenter hypervisors and hard partitioning hypervisors. The ACRN hypervisor architecture partitions the system into different functional domains, with carefully selected guest OS sharing optimizations for IoT and embedded devices. The __ACRN Device Model__ is a reference framework implementation for virtual device emulation that provides rich I/O virtualization support currently planned for audio, video, graphics, and USB. More mediator features are expected as the community grows.
+
+## Pre-requisites
+
+This document requires ACRN hypervisor and Kata containers installed on your system. The following provide links to installation of ACRN hypervisor and Kata containers:
+
+- ACRN supported [Hardware](https://projectacrn.github.io/latest/hardware.html#supported-hardware).
```

Comment:
Is the supported hardware relevant here, since the example is using the device model?

Reply:
This just tells users what hardware ACRN hypervisor and device model have been verified and found working. So it will be good to have this link.